testsupport/testng/src/main/java/com/adobe/testing/s3mock/testng/S3Mock.java

```diff
@@ -0,0 +1,72 @@
+/*
+ *  Copyright 2017-2018 Adobe.
+ *
+ *  Licensed under the Apache License, Version 2.0 (the "License");
+ *  you may not use this file except in compliance with the License.
+ *  You may obtain a copy of the License at
+ *
+ *          http://www.apache.org/licenses/LICENSE-2.0
+ *
+ *  Unless required by applicable law or agreed to in writing, software
+ *  distributed under the License is distributed on an "AS IS" BASIS,
+ *  WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ *  See the License for the specific language governing permissions and
+ *  limitations under the License.
+ */
+
+package com.adobe.testing.s3mock.testng;
+
+import com.adobe.testing.s3mock.S3MockApplication;
+import com.adobe.testing.s3mock.testsupport.common.S3MockStarter;
+
+import java.util.HashMap;
+import java.util.Map;
+
+public class S3Mock extends S3MockStarter {
```

Comment:
Please add some Javadoc, linking to `S3MockListener` as a requirement to use this class and maybe mentioning that it used as a singleton.